sdk/eventhubs/azure-messaging-eventhubs/inc/azure/messaging/eventhubs/event_data_batch.hpp

```diff
@@ -19,7 +18,7 @@ namespace Azure { namespace Messaging { namespace EventHubs {
   /** @brief EventDataBatchOptions contains optional parameters for the
    * [ProducerClient.CreateEventDataBatch] function.
    *
-   * @remark If both PartitionKey and PartitionID are nil, Event Hubs will choose an arbitrary
+   * @remark If both PartitionKey and PartitionId are nil, Event Hubs will choose an arbitrary
```

Comment:
Update go-ism of `nil` in comment. Is empty string the sentinel we use here?

```suggestion
   * @remark If both PartitionKey and PartitionId are empty, Event Hubs will choose an arbitrary
```